public/less/_repository.less

```diff
@@ -1583,6 +1583,10 @@
     display: none;
 }
 
+#issue-actions {
+  margin-top: -1rem !important; // counteract padding from Semantic
+}
+
```

Comment:
This comes from Semantic's `.ui .grid + .grid`. I was not able to overrule their rule with a `margin-top: 0` for some reason, so I went with this instead.